<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Two</class>
 <widget class="QWidget" name="Two">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1061</width>
    <height>626</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>490</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>返回</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>70</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="mylabel" name="label">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>440</y>
     <width>72</width>
     <height>141</height>
    </rect>
   </property>
   <property name="text">
    <string>图片1</string>
   </property>
  </widget>
  <widget class="mylabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>440</y>
     <width>72</width>
     <height>141</height>
    </rect>
   </property>
   <property name="text">
    <string>图片2</string>
   </property>
  </widget>
  <widget class="mylabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>450</y>
     <width>72</width>
     <height>121</height>
    </rect>
   </property>
   <property name="text">
    <string>图片3</string>
   </property>
  </widget>
  <widget class="mylabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>440</y>
     <width>72</width>
     <height>141</height>
    </rect>
   </property>
   <property name="text">
    <string>图片4</string>
   </property>
  </widget>
  <widget class="mylabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>450</y>
     <width>72</width>
     <height>131</height>
    </rect>
   </property>
   <property name="text">
    <string>图片5</string>
   </property>
  </widget>
  <widget class="mylabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>450</y>
     <width>72</width>
     <height>121</height>
    </rect>
   </property>
   <property name="text">
    <string>图片6</string>
   </property>
  </widget>
  <widget class="mylabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>440</y>
     <width>72</width>
     <height>131</height>
    </rect>
   </property>
   <property name="text">
    <string>图片7</string>
   </property>
  </widget>
  <widget class="mylabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>440</y>
     <width>72</width>
     <height>131</height>
    </rect>
   </property>
   <property name="text">
    <string>图片8</string>
   </property>
  </widget>
  <widget class="mylabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>440</y>
     <width>72</width>
     <height>151</height>
    </rect>
   </property>
   <property name="text">
    <string>图片9</string>
   </property>
  </widget>
  <widget class="mylabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>450</y>
     <width>72</width>
     <height>141</height>
    </rect>
   </property>
   <property name="text">
    <string>图片10</string>
   </property>
  </widget>
  <widget class="mylabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>440</y>
     <width>72</width>
     <height>141</height>
    </rect>
   </property>
   <property name="text">
    <string>图片11</string>
   </property>
  </widget>
  <widget class="mylabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>440</y>
     <width>72</width>
     <height>151</height>
    </rect>
   </property>
   <property name="text">
    <string>图片12</string>
   </property>
  </widget>
  <widget class="mylabel" name="label_13">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>440</y>
     <width>72</width>
     <height>151</height>
    </rect>
   </property>
   <property name="text">
    <string>图片13</string>
   </property>
  </widget>
  <widget class="mylabel" name="label_14">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>450</y>
     <width>72</width>
     <height>131</height>
    </rect>
   </property>
   <property name="text">
    <string>图片14</string>
   </property>
  </widget>
  <widget class="mylabel" name="label_15">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>450</y>
     <width>72</width>
     <height>131</height>
    </rect>
   </property>
   <property name="text">
    <string>图片15</string>
   </property>
  </widget>
  <widget class="mylabel" name="label_16">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>440</y>
     <width>72</width>
     <height>151</height>
    </rect>
   </property>
   <property name="text">
    <string>图片16</string>
   </property>
  </widget>
  <widget class="mylabel" name="label_17">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>450</y>
     <width>72</width>
     <height>141</height>
    </rect>
   </property>
   <property name="text">
    <string>图片17</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>360</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>出牌</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>360</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>不出</string>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>mylabel</class>
   <extends>QLabel</extends>
   <header location="global">mylabel.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
